pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -113,6 +113,7 @@ qtModule {
   ] ++ lib.optionals stdenv.isDarwin [
     "-DMAC_OS_X_VERSION_MAX_ALLOWED=MAC_OS_X_VERSION_10_12"
     "-DMAC_OS_X_VERSION_MIN_REQUIRED=MAC_OS_X_VERSION_10_12"
+    "-Wno-elaborated-enum-base"
```

Comment:
Should this be `stdenv.isClang`?

Reply:
Yes, it should : )

Reply:
I rebased on a more recent staging-next commit in preparation of more evaluations. Hope it doesn't mess up reviews.